src/Microsoft.DotNet.VersionTools/Util/Command.cs

```diff
@@ -0,0 +1,349 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.VersionTools.Util
+{
+    class Command
```

Comment:
Yes, let's not make this library dependent on MSBuild.

It appears that there are only a handful of places this is called - just the "git" commands right?  I don't think making the caller invoke the "git" commands is a good approach, that's a lot of benefit of having this logic in a centralized place.

All that the code needs is a way to start a few processes: git commit, push, status.